### PR TITLE
Docs: IoT docs copyedits

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: documentation/modules/proc-iot-creating-project.adoc [hunk 1]
```diff
@@ -7,7 +12,7 @@ ifeval::["{cmdcli}" == "oc"]
 . Change to a non-admin user:
```
Comment (jenmalloy) [inline documentation/modules/proc-iot-creating-project.adoc]: @dejanb Seems like the command is missing here? Or maybe the colon should just be a period here (if it depends what the exact command will be)

Comment (dejanb) [reply documentation/modules/proc-iot-creating-project.adoc]: @jenmalloy You're right ... it should be the same as Step 1 in https://enmasse.io/documentation/master/openshift/#create-address-space-cli-messaging-gs (I can't link to it in github properly).

**Comment by jenmalloy** [comment]
@dejanb, Could you review the latest changes please? Thanks!

File: documentation/modules/proc-iot-api.adoc [hunk 2]
```diff
@@ -13,17 +18,22 @@ endif::[]
 
 = Sending and receiving {api} data
 
-.Procedure
+== Starting the {api} consumer
 
-== Starting {api} consumer
+.Prerequisites
```
Comment (bhardesty) [inline documentation/modules/proc-iot-api.adoc]: Just a minor suggestion for prereqs: I typically state them as present conditions (so, "The IoT services are installed", "An IoT project is created", etc.)

However, if the other prereqs in this doc are phrased a certain way, I'd just match it here for consistency.

File: documentation/modules/proc-iot-api.adoc [hunk 3]
```diff
@@ -13,17 +18,22 @@ endif::[]
...
 
-. In Hono project run:
+. In the Hono project, run:
 +
 [options="nowrap",subs="attributes"]
 ----
 cd cli
 ----
-. At least once run:
+. At least once, run:
```
Comment (bhardesty) [inline documentation/modules/proc-iot-api.adoc]: Maybe something like, "Run the following command at least once:"

Still, it's not clear what "at least once" means. How do I know if I need to run it more than once?

File: documentation/modules/proc-iot-creating-project.adoc [hunk 4]
```diff
@@ -1,13 +1,28 @@
...
+//
+// assembly-iot-guide.adoc
+// assembly-IoT.adoc
+
 [id='iot-creating-project-{context}']
 = Creating an IoT project
 
+After installing the IoT services, you create an IoT project.
+
+.Prerequisite
```
Comment (bhardesty) [inline documentation/modules/proc-iot-creating-project.adoc]: ".Prerequisites" should be plural even if there's only one prereq.

**Pull request merged**